cgsmiles/tests/test_cgsmile_parsing.py

```diff
@@ -318,16 +318,16 @@ def test_read_cgsmiles(smile, nodes, charges, edges, orders):
                         None,
                         None),
                         # simple chirality in residue
-                        ("[>]C[C@](F)(B)N[<]",
+                        ("[>]C[C@R](F)(B)N[<]",
                         "C[C](F)(B)N",
                         {0: [">1"], 4: ["<1"]},
-                        {1: ('@', [])},
+                        {1: '@R'},
```

Comment:
`@R` or just `R`?

Reply:
I deliberately chose `@R` to stress that this is a chirality assignment.